app/models/framework/management_charge_calculator/flat_rate.rb

```diff
@@ -7,6 +7,8 @@ module ManagementChargeCalculator
     # calculated against another column, that can be specified with the `column`
     # attribute.
     class FlatRate
+      NUMBER_WHITELIST = /([^0-9.\-]+)/.freeze
```

Comment:
If I were the nitpicky type, I'd point out that this is more of blacklist / exclusion list than a whitelist / inclusion list...

Reply:
👍🏽